tests/test_pluggable.py

```diff
@@ -630,7 +630,9 @@ def test_retrieve_subject_token_missing_error_code_message(self):
             )
 
     @mock.patch.dict(os.environ, {"GOOGLE_EXTERNAL_ACCOUNT_ALLOW_EXECUTABLES": "1"})
-    def test_retrieve_subject_token_missing_expiration_time(self):
+    def test_retrieve_subject_token_without_expiration_time_should_fail_when_using_output_file(
```

Comment:
when_using_output_file is a little bit misleading because here we're reading from STDOUT.

maybe 'when_output_file_specified'?